### ISSUE TITLE
some confusions

### ISSUE THREAD

**Issue by sjjlr** [openclimatefix/skillful_nowcasting#6]
1, Should the original radar data be normalized to [0,1]？The paper said the radar data was transformed to rain rate, then is the rain rate be normalized to [0,1] or [-1, 1]？
2, The output channel of conditioning stack is not consistent with the code, for example, for the last layer, 384 vs 768, while the code cannot work when channel is 384.
3, The predicted radars are very similar as the last frame of input data, but without any cloud movement. I don't know why. It seems that the convGRU doesnot work.

**Comment by jacobbieker** [comment]
I did just swap out the ConvGRU implementation with the one from our implementation of MetNet, and fixed a few errors, so as long as you run the #5 branch, it should at least run, although results are still probably not great

**Comment by sjjlr** [comment]
Just because GAN is hard to train? Training stablely is one thing, I don't understand why there is no cloud movement using ConvGRU. There must be something wrong. In my opinion, even if the training is unstable, the cloud should move, although the movement is in wrong direction.

**Comment by jacobbieker** [comment]
One challenge we have faced is having large enough batch sizes and training long enough to get the model to learn anything. I don't quite remember what the authors' reasoning was, but in our own attempts at getting it working, we've been a bit constrained on getting large enough batches and parameters to train it. I'm hoping by using the weights from the official implementation, we can get it working well enough that fine-tuning it might work.

**Comment by sjjlr** [comment]
I notice that there are some differences between the forward function of ConvGRUCell and the original ConvGRU paper, can you check it? 

meanwhile, if the model is trained using 256*256 croped images, how does it work on the whole 1500*1200 images? croping and mosaicing? or the whole 1500*1200 images can be input into the model directly?

**Comment by jacobbieker** [comment]
They did just add the psuedocode for almost the whole model now, see the comments in #5, so I now have their implementation of the ConvGRU, etc. So soon this repo will be a nearly exact PyTorch port of their model, and hopefully be able to load their weights quite easily.

**Comment by jacobbieker** [comment]
Now that #5 is merged, all the layers in the model  are setup as close as possible to the pseudocode, so should work the same. There is an issue with #10 but otherwise, it should work the same.

**Comment by john970** [comment]
I know this is an old and closed issue, but I wanted to confirm this was not supposed to be the case with the latest code? I am seeing the same issue with the pretrained model.
